test/performance/odp_l2fwd.c

```diff
@@ -2210,7 +2212,13 @@ int main(int argc, char *argv[])
 
 	bind_workers();
 
-	odp_schedule_config(NULL);
+	odp_schedule_config_init(&sched_config);
+	if (odp_schedule_capability(&sched_capa)) {
+		ODPH_ERR("Error: schedule capability failed\n");
+		exit(EXIT_FAILURE);
+	}
+	sched_config.max_flow_id = sched_capa.max_flow_id;
+	odp_schedule_config(&sched_config);
```

Comment:
A user may also be interested about performance without flow aware scheduling. This change would also break a rather long performance measurement record we have with this application. So, it would be better to put this behind a configuration option (e.g. `-f, --flow_aware`) and keep the default value as disabled.

Reply:
Sure. I will post the next version with the change.